engine/src/main/scala/cromwell/engine/backend/local/SharedFileSystemBackend.scala

```diff
@@ -23,31 +21,41 @@ import scala.util.{Failure, Success, Try}
 object SharedFileSystemBackend {
   type LocalizationStrategy = (String, Path, WorkflowDescriptor) => Try[Unit]
 
-  private def localizeFromGcs(originalPath: String, executionPath: Path, descriptor: WorkflowDescriptor): Try[Unit] = Try {
+  private[local] def localizeFromGcs(originalPath: String, executionPath: Path,
```

Comment:
@Horneth I was reading your bit as meaning that if this were being developed in the not too distant future it should be done differently due to changes you're implementing. If that's the case it's really just needs to be a reminder w/ enough info that people will know what it means, or if I misunderstood then forget about it :)